### PR TITLE
LangSplatv2: Implement mask + CLIP feature transforms

### LINKED ISSUE #31: Implement CLIP feature transform
Implement a dataset transform that runs inference of the CLIP foundation model to generate features for a dataset in a similar style to how SAM2 is currently utilized to generate mask data for the instance segmentation models.

### PR DESCRIPTION
**Pull request by swahtz**
Implements the preprocessing pipeline to generate the SAM2 masks and CLIP feature generation as transforms in the same style as garfvdb's implementation.
closes #31

### CODE REVIEW

File: open_vocabulary_segmentation/langsplatv2/pyproject.toml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+[build-system]
+requires = ["setuptools>=61.0", "wheel"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "fvdb-langsplatv2"
+version = "0.0.1"
+description = "fVDB Implementation of LangSplatV2"
+readme = "README.md"
```
[the remark this answers is not in the document]
Comment (swahtz) [reply open_vocabulary_segmentation/langsplatv2/pyproject.toml]: Done

File: open_vocabulary_segmentation/langsplatv2/langsplatv2/scene_transforms/clip_feature_encoding.py [hunk 2]
```diff
@@ -0,0 +1,372 @@
...
+        Args:
+            image: Source image in RGB format, shape [H, W, 3].
+            masks: Binary masks, shape [N, H, W].
+            bboxes: Bounding boxes in XYWH format, shape [N, 4].
+
+        Returns:
+            CLIP embeddings for each masked region, shape [N, clip_n_dims].
+        """
+        if len(masks) == 0:
+            return torch.zeros(0, self._clip_n_dims)
```
[the remark this answers is not in the document]
Comment (swahtz) [reply open_vocabulary_segmentation/langsplatv2/langsplatv2/scene_transforms/clip_feature_encoding.py]: Done

File: open_vocabulary_segmentation/langsplatv2/langsplatv2/config.py [hunk 3]
```diff
@@ -0,0 +1,244 @@
...
+
+    # Image preprocessing
+    image_downsample_factor: int = 1
+    """Factor by which to downsample images before processing."""
+
+    rescale_jpeg_quality: int = 95
+    """JPEG quality (0-100) when resaving downsampled images."""
+
+    max_image_resolution: int = 1080
+    """Maximum image height; larger images will be downsampled."""
```
[the remark this answers is not in the document]
Comment (swahtz) [reply open_vocabulary_segmentation/langsplatv2/langsplatv2/config.py]: Done
